core/monitor/auction.go

```diff
@@ -63,9 +63,9 @@ func NewAuctionState(mkt *types.Market, now time.Time) *AuctionState {
 
 // StartLiquidityAuction - set the state to start a liquidity triggered auction
 // @TODO these functions will be removed once the types are in proto.
-func (a *AuctionState) StartLiquidityAuction(t time.Time, d *types.AuctionDuration) {
+func (a *AuctionState) StartLiquidityAuction(t time.Time, d *types.AuctionDuration, tigger types.AuctionTrigger) {
```

Comment:
Passing the trigger type in fits badly with the other functions. I can currently call it like this:
```
StartLiquidityAuction(types.AuctionTriggerPrice)
```
and it'll set the state to be a price auction, despite the function being called `StartLiquidityAuction()`.

We should either have one general function that takes in a trigger for all cases, or split `StartLiquidityAuction()` into two.

Reply:
Also a typo: `tigger` -> `trigger`.